apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -127,7 +127,9 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
             return location && location.title && location.title !== '';
           }
         ),
-      imageFile: Yup.mixed().required('Image is required.'),
+      imageFile: Yup.mixed()
+        .required('Image is required.')
+        .test('big-file', 'File Size Too Large', validateFileSize),
```

Comment:
Error message should not be in camel case to match other error messages. 
```suggestion
        .test('big-file', 'File size is too large.', validateFileSize),
```

Reply:
Done